### ISSUE TITLE
Cannot start nuxt:  nuxtCtx.tryUse is not a function

### ISSUE THREAD

**Issue by OyewoleOyedeji** [gitFoxCode/nuxt-icons#8]
When I added `nuxt-icons` to the modules array in the `nuxt.config.js` I get the following error while and after server restart

```log
ERROR  Cannot start nuxt:  nuxtCtx.tryUse is not a function                              16:58:59

  at useNuxt (node_modules/nuxt-icons/node_modules/@nuxt/kit/dist/index.mjs:47:28)
  at addTemplate (node_modules/nuxt-icons/node_modules/@nuxt/kit/dist/index.mjs:58:16)
  at setup (node_modules/nuxt-icons/dist/module.mjs:18:5)
  at Object.normalizedModule (node_modules/nuxt-icons/node_modules/@nuxt/kit/dist/index.mjs:624:29)
  at async installModule (node_modules/@nuxt/kit/dist/index.mjs:432:3)
  at async initNuxt (node_modules/nuxt/dist/index.mjs:1362:7)
  at async load (node_modules/nuxi/dist/chunks/dev.mjs:6750:9)
  at async Object.invoke (node_modules/nuxi/dist/chunks/dev.mjs:6795:5)
  at async _main (node_modules/nuxi/dist/cli.mjs:46:20)
```

**Comment by gitFoxCode** [comment]
Could you write more about your environment in which the error occurs? Version of nuxt, operating system etc.

**Comment by durbanitas** [comment]
I got the same error with Nuxt `3.0.0-rc.4`

**Comment by OyewoleOyedeji** [comment]
@gitFoxCode Sorry for the late reply.

I use the latest version of the `Nuxt 3 Release Candidate`

and

the operating system Manjaro [Arch]

**Comment by OyewoleOyedeji** [comment]
@gitFoxCode my environment for the project is

------------------------------
- Operating System: `Linux`
- Node Version:     `v16.16.0`
- Nuxt Version:     `3.0.0-rc.1`
- Package Manager:  `yarn@1.22.19`
- Builder:          `vite`
- User Config:      `typescript`, `modules`, `app`, `buildModules`, `googleFonts`
- Runtime Modules:  `@nuxtjs/tailwindcss@5.3.2`
- Build Modules:    `@nuxtjs/google-fonts@3.0.0-0`
------------------------------

**Comment by gitFoxCode** [comment]
@OyewoleOyedeji  @durbanitas  The latest version, available on the newly released stable nuxt 3.0.0 no longer contains such bugs
